models/Document/PageSnippet.php

```diff
@@ -108,6 +108,31 @@ abstract class PageSnippet extends Model\Document
      */
     protected $inheritedEditables = [];
 
+    /**
+     * @var bool
+     */
+    private static $getInheritedValues = false;
+
+    /**
+     * @static
+     *
+     * @param bool $getInheritedValues
+     */
+    public static function setGetInheritedValues($getInheritedValues)
+    {
+        self::$getInheritedValues = $getInheritedValues;
+    }
+
+    /**
+     * @static
+     *
+     * @return bool
+     */
+    public static function getGetInheritedValues()
+    {
+        return self::$getInheritedValues;
+    }
```

Comment:
We should use typehints instead of php docs.
```suggestion
    private static bool $getInheritedValues = false;

    public static function setGetInheritedValues(bool $getInheritedValues): void
    {
        self::$getInheritedValues = $getInheritedValues;
    }

    public static function getGetInheritedValues(): bool
    {
        return self::$getInheritedValues;
    }
```

Reply:
ah,right.. i just copied without considering that 🤦🏻 
Thank you again!